randovania/gui/ui_files/games_tab_planets_zebeth_widget.ui

```diff
@@ -30,15 +30,12 @@
       <item>
        <widget class="QLabel" name="intro_label">
         <property name="text">
-         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
-&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
-p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Traverse Zebes and its depths and kill Kraid and Ridley to fight Mother Brain and escape Zebes.&lt;/p&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Two statues will block the path to Tourian, unless you killed both Kraid and Ridley. You can find them in their own lair.&lt;/p&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Some hints are added to locate different pickups but only ice beam and missiles are required.&lt;/p&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;For more details, check out the &lt;a href=&quot;tab://FAQ&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;FAQ&lt;/span&gt;&lt;/a&gt; and &lt;a href=&quot;tab://Differences&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Differences&lt;/span&gt;&lt;/a&gt; tabs.&lt;/p&gt;
-&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To get started, use the Quick Generate button to generate a game using the default settings!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Traverse Zebes and its depths and kill Kraid and Ridley to fight Mother Brain and escape Zebes.&lt;/p&gt;
```

Comment:
This sentence reads a tad weird with how many `and`s it contains. 
`Traverse the depths of Zebes, to kill Kraid and Ridley in order to fight Mother Brain and Escape Zebes.`?

Reply:
it's outdated btw

Reply:
comment still applies, even the current text contains too many `and`s for my liking.

Reply:
Changed it a bit

Reply:
![image](https://github.com/randovania/randovania/assets/14176397/76089eb6-ccc7-4406-90b9-206f0a9a7598)

Reply:
would still use `Traverse the depths of Zebes` TBH

Reply:
Done